Building a ES2015-based app fails

When doing `grunt` or `grunt build`, the `eslint` accuses things like this:

> app/scripts/main.js
  1:1  error  Unexpected token let

This is "fixed" by changing the folder `eslint` is searching for do the lint, like from `<%= config.app %>/scripts/{,*/}*.js` to `<%= config.tmp %>/scripts/{,*/}*.js`. This is happening obviously because it is searching within the `app` directory, which has the non-compiled ES2015 code and eslint doesn't know much about it, unless you defined `environment: 'es6'`, but both cases leads to errors like this:

> .tmp/scripts/main.js
    3:13  error  "Backbone" is not defined

Also uglifying is not working as well. It is attempting to do the same as eslint: doing its task from ES2015 code, which has an unknown syntax for it.

Two alternatives I found so far:

1. Uglifying manually;
2. Avoid ES2015—which this generator is proposed to handle.

By the bye, do you have any suggestions?

--- a/app/templates/index.html
+++ b/app/templates/index.html
@@ -91,7 +91,7 @@
     <!-- endbuild -->
     <% } %>
 
-    <!-- build:js scripts/main.js -->
+    <!-- build:js(.tmp) scripts/main.js -->
     <script src="scripts/main.js"></script>
     <!-- endbuild -->
 

